### PR TITLE
Update configure-email-forwarding.md

### PR DESCRIPTION
**Pull request by msbemba**
Clarified admin permissions

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Office365-Admin/email/configure-email-forwarding.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ As the admin of an Office 365 organization, you might have company requirements
     
 - Email forwarding requires that the  *from*  account has a license. If you're setting up email forwarding because the user has left your organization, another option is to [convert their mailbox to a shared mailbox](convert-user-mailbox-to-shared-mailbox.md). This way several people can access it. However, a shared mailbox cannot exceed 50GB. 
     
-You must have [admin permissions](../add-users/about-admin-roles.md) in Office 365 to do these steps. 
+You must be an Exchange administrator or Global administrator in Office 365 to do these steps. For more information check [admin permissions](../add-users/about-admin-roles.md)
```
Comment (JohanFreelancer9) [inline Office365-Admin/email/configure-email-forwarding.md]:
```suggestion
You must be an Exchange administrator or Global administrator in Office 365 to do these steps. For more information, see the topic [About admin roles](../add-users/about-admin-roles.md).
```

**Comment by mypil** [comment]
@msbemba - please merge the commit on the copy/edit review done by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by mypil** [comment]
@cmcatee-MSFT  - PR has been copy edited and is ready for final review. This is based on issue #943 

Thank you.

cc: @kenwith

**Pull request merged**